leaflet-amplify/README.md

```diff
@@ -0,0 +1,48 @@
+# Leaflet AWS Amplify x Amazon Location Service
+
+![](./img/README01.gif)
+
+![](./img/README02.png)
+
+This example demonstrates how to use Leaflet with Amazon Location Service and AWS Amplify.
+
+## Execution environment
+
+- node v18.x
+- npm v8.x
+- Amplify CLI v9.2.x
```

Comment:
You have used version v10.x to create the Amplify resources according to the config file in the `amplify` folder.

I'm not 100% sure if they're compatible, so either:
- Change the min version in the readme to match the one you have used.
- OR test the sample with Amplify CLI 9.2.x and confirm it works.

Reply:
Fixed.